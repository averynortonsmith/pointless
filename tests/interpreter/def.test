
output = [a, b, c, d, x] |> println

x = 123
(a, b) = (false, Foo)
(c, d) = Bar(false, Foo)