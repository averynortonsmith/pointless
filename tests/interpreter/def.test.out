[false, Foo, false, Foo, 123]
